app/Transformers/BeatmapsetTransformer.php

```diff
@@ -59,6 +59,7 @@ public function transform(Beatmapset $beatmapset = null)
             'tags' => $beatmapset->tags,
             'video' => $beatmapset->video,
             'status' => $beatmapset->status(),
+            'ratings' => $beatmapset->ratingsCount(),
```

Comment:
Also, you don't want this here - it will slow down every other page that uses the BeatmapsetTransformer... especially ones that display multiple Beatmapsets (e.g. user profile page)

I'd move it to an include and only use it when required instead.